remoteappmanager/handlers/home_handler.py

```diff
@@ -254,7 +252,6 @@ def _start_container(self, orm_user, app, policy, mapping_id):
         Container
         """
 
-        user_name = orm_user.name
```

Comment:
The fact that we use only one part of an object does not imply that you should just pass that part. The API is made so that it homogeneously accepts objects. Now it's no longer so.